covalent/_workflow/transport.py

```diff
@@ -439,6 +440,47 @@ def get_internal_graph_copy(self) -> nx.MultiDiGraph:
 
         return self._graph.copy()
 
+    def reset_node(self, node_id: int) -> None:
+        """Reset node values to starting state."""
+        self.set_node_value(node_id, "start_time", None)
```

Comment:
You can probably use a list of strings here then iterate over it since most of them are being set to `None` and handle `status` outside of the loop.